New ChatIntegration functionality RN35983 (#1229)
SkylineCommunications/dataminer-docs: user-guide/Cloud_Platform/TeamsBot/Microsoft_Teams_Chat_Integration.md

## Limitations

> - Currently, you cannot send messages in fetched channels where the DataMiner Teams bot wasn't installed in the team. This will be indicated in the error when you try to send a channel notification. You can [manually install the DataMiner Teams bot in the team using Microsoft Teams](xref:DataMiner_Teams_bot#dataminer-teams-bot-installation) or use the create team functionality of chat integration to create another team with the DataMiner Teams bot automatically installed.

Reviewer: ```suggestion
- Currently, it is not possible to send messages in fetched channels for which the DataMiner Teams bot is not installed in the team. This will be indicated in the error when you try to send a channel notification. You can [manually install the DataMiner Teams bot in the team using Microsoft Teams](xref:DataMiner_Teams_bot#dataminer-teams-bot-installation) or use the create team functionality of chat integration to create another team with the DataMiner Teams bot automatically installed.
```

Author: I'll try to fix this limitation

Author: done
